tests/execution_engine/conftest.py

```diff
@@ -26,7 +25,7 @@ def aws_credentials():
 @pytest.fixture
 def s3(aws_credentials):
     with mock_s3():
-        yield boto3.client("s3", region_name="us-east-1")
+        yield aws.boto3.client("s3", region_name="us-east-1")
```

Comment:
@Shinnnyshinshin Will this not fail if `aws` "is not installed"?..  Thanks.

Reply:
This one is taken care of by the `pyskip`. We will not load/run the fixture if `aws` doesn't exist.. so it's caught earlier than this line